protocol/avdl/stellar1/local.avdl

```diff
@@ -67,8 +67,8 @@ protocol local {
   record PaymentLocal {
     PaymentID id;                   // For relay payments, the funding tx.
     TimeMs time;
-    PaymentStatus statusSimplified; // we are working on making this correct for relay payments...
-    string statusDescription;       // "pending", "completed", "error"
+    PaymentStatus statusSimplified;
+    string statusDescription;       // "pending", "completed", "error", "claimable", "canceled"
```

Comment:
It would be cleaner to just have PaymentStatus. Is there a reason to have duplicate fields?

Reply:
@mlsteele / @zanderz I'm fine with removing it, there's some more JS changes too so I'd rather do it separately

Reply:
Separately sounds great